umap/static/umap/js/modules/utils.js

```diff
@@ -416,9 +416,11 @@ export function loadTemplate(html) {
 }
 
 export function loadTemplateWithRefs(html) {
-  const element = loadTemplate(html)
+  const template = document.createElement('template')
```

Comment:
I don't get the difference with `loadTemplate`?

Reply:
We need to run the querySelectorAll on the content element, so if the root element itself has a `data-ref` it will be taken into account.